### ISSUE TITLE
[Accessibility: ACC-264.1] Input field, Combo box, Multi Combo box, Date Picker, VHD: Labels and required info not read

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#12568]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
1. The corresponding label and group label should be read out, when each & every UI is accessed.
2. Required information should be read out for all the mandatory fields.

<img width="452" alt="image" src="https://github.com/user-attachments/assets/da22241d-be3d-4e32-8cf0-e4acb9009b4f">
<img width="452" alt="image" src="https://github.com/user-attachments/assets/9494029a-6092-4ed7-93a0-205d39fba59e">
<img width="452" alt="image" src="https://github.com/user-attachments/assets/07fb365a-1d9e-4075-a40d-02460eb4a9ae">

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All versions.

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Multiple component has same issues:
1. [Input](https://sap.github.io/fundamental-ngx/#/core/input)
2. [Combobox](https://sap.github.io/fundamental-ngx/#/core/combobox)
4. [Multi Combobox](https://sap.github.io/fundamental-ngx/#/core/multi-combobox)
5. [Date Picker](https://sap.github.io/fundamental-ngx/#/core/date-picker)
6. [VHD](https://sap.github.io/fundamental-ngx/#/platform/value-help-dialog)

To reproduce:
Use any of the above component and access through screen reader.
Labels and required information not read throughout the application

#### Please provide stackblitz example(s).
You can try ngx examples, issue is reproducible there.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes, issue is raised by an accessibility expert.

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
@mikerodonnell89 Can you check that?

**Comment by mikerodonnell89** [comment]
@divyamalhotra488 this appears to have been fixed - please reopen a new issue with more details on how to reproduce (perhaps a public stackblitz demonstrating the issue) if the problem persists

<img width="1371" alt="Image" src="https://github.com/user-attachments/assets/514f252d-edbd-4f70-83f4-4fba37f95ad4" />
